pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -1904,6 +1991,39 @@ async def _get_event_iterator(self) -> AsyncIterator[ModelResponseStreamEvent]:
                 )
                 yield self._parts_manager.handle_part(vendor_part_id=f'{chunk.item_id}-file', part=file_part)
 
+            elif isinstance(chunk, responses.ResponseMcpCallArgumentsDoneEvent):
+                pass  # there's nothing we need to do here
+
+            elif isinstance(chunk, responses.ResponseMcpCallArgumentsDeltaEvent):
+                try:
+                    json_delta = json.loads(chunk.delta)
```

Comment:
I don't think we should need to do this. If the delta is partial JSON, can't we pass it directly to `handle_tool_call_delta` as `args`?